### DISCUSSION TITLE
lc/91/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2436] [Announcements]
# lc/91/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/91/

**Comment by havenBoy** [comment]
f(i)单独成解法前提是f(i-1)!='0' 那么对于206这种字符串，不应该拆分为20 与6  ，总共解法为2个吗？

**Reply by yanglbme** [reply]
拆分为 20 和 6，总共解法是 1
